FreeRTOS/FreeRTOS
# Fixed some typos in the readme in the Posix_GCC demo

Fixing some typos in a readme

Description
-----------
"profilig" -> "profiling"
"possix" -> "posix"


Test Steps
-----------
/

Related Issue
-----------
/


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
